pkg/rtc/room.go

```diff
@@ -199,7 +288,7 @@ func NewRoom(
 
 	r.createAgentDispatchesFromRoomAgent()
```

Comment:
can this be async?

Reply:
We probably want createAgentDispatchesFromRoomAgent to have run by the time NewRoom returns, as its output is needed by the agents lifecycle RPCs. It does very little though (the only potentially blocking call is to storage). 

launchRoomAgents could be run in its own goroutine, but it already instantiate goroutines internally do do actual work